homeassistant/components/powerwall/__init__.py

```diff
@@ -115,13 +115,28 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     await _migrate_old_unique_ids(hass, entry_id, powerwall_data)
     login_failed_count = 0
 
+    entry_data = hass.data[DOMAIN][entry.entry_id] = {
```

Comment:
entry_data is a bit of a confusing name, because there is also `entry.data`

Reply:
Yeah. Its kinda generic. Any suggestions on a better one?  I ask because its present in a lot of other integrations as well.

Reply:
Not sure, maybe `runtime_data` or `setup_data` ? Or maybe just `data`

Reply:
Will adjust pending the outcome of the testing 👍